scripts/tmux-user-shell.sh

```diff
@@ -0,0 +1,41 @@
+sleep 2
+
+POLL_INTERVAL=0.5
+
+# Initialize wallet and get ourselves some money
+function mine_blocks() {
+    PEG_IN_ADDR="$($FM_BTC_CLIENT getnewaddress)"
+    $FM_BTC_CLIENT generatetoaddress $1 $PEG_IN_ADDR
+}
+
+function open_channel() {
```

Comment:
I made a `lib.sh` file because `build.sh` is called outside the `tmux` environment and I can't really carry anything over (see the hack to get the env vars transferred).